conda-environment.yml

```diff
@@ -24,3 +24,4 @@ dependencies:
   - nodejs ==18.*|20.*  # Make sure it matches `ui/package.json`
   - pnpm ==9.*  # Make sure it matches `ui/package.json`
   - poetry
+  - ffmpeg  # Not directly used by mxcube, but necessary for the video-streamer
```

Comment:
1. I recommend using alphabetical order within each section.
2. ffmpeg is a runtime dependency not a development dependency, so it should be listed in the "Runtime dependencies" section

Reply:
You are right, I will change this. Thanks for pointing it out!